spec/defines/vhost_spec.rb

```diff
@@ -85,6 +85,7 @@
               'ssl_proxyengine'             => true,
               'ssl_proxy_cipher_suite'      => 'HIGH',
               'ssl_proxy_protocol'          => 'TLSv1.2',
+              'ssl_user_name'               => 'SSL_CLIENT_S_DN_CN',
```

Comment:
where is the verification of this spec addition?

Reply:
It looks like a lot of the SSL options aren't actually verified https://github.com/puppetlabs/puppetlabs-apache/blob/bd7942973532003ee00c8a3e32fc4399ef75ee28/spec/defines/vhost_spec.rb#L1041

I'll add something for this.

Reply:
Done.